### DISCUSSION TITLE
Issue with Find-FalconDuplicate

### DISCUSSION THREAD

**Discussion by Wonkeysmoker** [CrowdStrike/psfalcon#215] [Q&A]
Good Day,

I am successfully authenticated and psfalcon is functioning. I just completed using Invoke-FalconHostGroupAction -Name hide-host -for 
 devices not seen in > 30 days.
 
I then tried to use Find-FalconDuplicate | Export-Csv C:\temp\duplicates.csv 

the csv gets created but it stays at 0kb and the command never finishes. I only download psfalcon today so it is version 2.18. any suggestions?

**Comment by bk-cs** [comment]
v2.1.8 was released in March 2022, and the current version of PSFalcon is v2.2.0. Please remove any existing installations of PSFalcon and re-install using the recommended [PowerShell Gallery instructions](https://github.com/CrowdStrike/psfalcon/wiki/Installation,-Upgrade-and-Removal#use-the-powershell-gallery).

If you still have trouble after a re-installation, [please open an issue](https://github.com/CrowdStrike/psfalcon/issues/new/choose) and include the PowerShell transcript created after following these steps:
```
> Import-Module -Name PSFalcon
> Request-FalconToken <with relevant parameters>
> $VerbosePreference = 'Continue'
> Start-Transcript
> Find-FalconDuplicate | Export-Csv C:\temp\duplicates.csv

<Press Ctrl-C to end command, if necessary>

> Stop-Transcript
```
